desc/vmec_utils.py

```diff
@@ -468,3 +483,461 @@ def vmec_boundary_subspace(eq, RBC=None, ZBS=None, RBS=None, ZBC=None):  # noqa:
     boundary_subspace = block_diag(Rb_subspace, Zb_subspace)
     opt_subspace = null_space(boundary_subspace)
     return opt_subspace
+
+
+def make_boozmn_output(  # noqa: C901
+    eq, path, surfs=128, M_booz=None, N_booz=None, verbose=0
+):
+    """Create and save a booz_xform-style .nc output file.
+
+    based strongly off of https://github.com/hiddenSymmetries/booz_xform/tree/main
+
+    Parameters
+    ----------
+    eq : Equilibrium
+        Equilibrium to save.
+    path : str
+        File path of output data.
+    surfs: int
+        Number of flux surfaces to calculate Boozer transform at (Default = 128).
+        NOTE: because this is performed on the so-called "half-grid", this will
+        result in an output of size surfs-1, since for surfs number of surfaces
+        there is only surfs-1 surfaces on the half-grid, the first one being
+        the surface at s = 0.5 / surfs
+        where s = rho**2 is the normalized toroidal flux coordinate
+    M_booz : int, optional
+        poloidal resolution to use for Boozer transform. Default is `2 * eq.M`.
+    N_booz : int, optional
+        toroidal resolution to use for Boozer transform.Default is `2 * eq.N`.
+    verbose: int
+        Level of output (Default = 1).
+        * 0: no output
+        * 1: status of quantities computed
+        * 2: as above plus timing information
+
+    Returns
```

Comment:
add a warning if eq is asym that `nu` is opposite sign of that saved from SIMSOPT and thus not verified, though every other quantity is checked and agrees with the other versions.